listenbrainz/webserver/static/js/src/LastFMImporterModal.tsx

```diff
@@ -1,6 +1,7 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Do we have a typescript equivalent of this rule? I'd just prefer to disable it everywhere